File: CCp/TypeScript/src/Math/score.ts

// X is the total points for the problem
// A Problem consists of 10 testcases
// Y is the testcases chef has passed

function chefPoints ( X: number, Y: number ) {

  const pointsForEachTestCase: number = ( X / 10 );
  console.log( pointsForEachTestCase );
  console.log( Y * pointsForEachTestCase );
}

chefPoints( 130, 4);

File: CCp/C/Math/score.c
#include <stdio.h>

int main(void)
{

  int noOfTestCases, totalScore, testsCasesPassed;

  printf("Enter the no of test cases ");
  scanf("%d", &noOfTestCases);

  for (int i = 1; i <= noOfTestCases; i++)
  {

    printf("please enter total score and the no of testcases chef has passed ");
    scanf("%d%d", &totalScore, &testsCasesPassed);

    int pointsPerTestCase = totalScore / 10;
    printf("%s %d %s \n", "chef has scored ", (testsCasesPassed * pointsPerTestCase), " points ");
  }
}